modules/base/rendering/renderabletrail.cpp

```diff
@@ -293,7 +293,7 @@ void RenderableTrail::internalRender(bool renderLines, bool renderPoints,
     // is necessary as they are usually far away from their reference
     _programObject->setUniform(
         _uniformCache.modelView,
-        data.camera.combinedViewMatrix() * modelTransform * info._localTransform
+        calcModelViewTransform(data, modelTransform) * info._localTransform
```

Comment:
How about saving in a variable instead? To avoid multiple operations on the same line